### PR TITLE
Juju export bundle cli forward port

### PR DESCRIPTION
**Pull request by vinu2003**
## Description of change
This PR brings in export-bundle CLI code changes to develop branch.
This also includes feature tests added.
## QA steps
execute 'juju export-bundle'
## Documentation changes
n/a
workflow? CLI? API?* 
n/a
## Bug reference
n/a

### CODE REVIEW

File: cmd/juju/model/exportbundle.go [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+const exportBundleHelpDoc = `
+Exports the current model configuration as bundle.
+
+If --filename is not used, the configuration is printed to stdout.
+ --filename specifies an output file.
+
+Examples:
+
+    juju export-bundle
+	juju export-bundle --filename mymodel
```
Comment (anastasiamac) [inline cmd/juju/model/exportbundle.go]: Indentation is different between the 2 lines in the Examples section.

**Comment by anastasiamac** [comment]
You might want to address featuretest as well - http://ci.jujucharms.com/job/github-check-merge-juju/2631/testReport/junit/github/com_juju_juju_featuretests/TestPackage/

File: cmd/juju/model/exportbundle.go (hunk 1 above)
Comment (vinu2003) [reply cmd/juju/model/exportbundle.go]: done

**Comment by vinu2003** [comment]
Hi @anastasiamac, Whether it is develop - mode command or not. Good question. But i should be confirming that either with Tim or Ian.

The feature test failure is because, the expected reflects the fix i made in server code - removing bindings attr from application. But this fix is not yet in develop branch. 

Lets merge this after the confirmation
Thank you.

**Pull request closed without merging**